catalog/dags/providers/provider_api_scripts/flickr.py

```diff
@@ -120,9 +120,9 @@ def ingest_records(self, **kwargs):
         for start_ts, end_ts in self.large_batches:
             # For each large batch, ingest records for that interval one license
             # type at a time.
-            for license in LICENSE_INFO.keys():
+            for license_ in LICENSE_INFO.keys():
```

Comment:
Just for my own edification, what is the reason behind this rename?

Reply:
Pycharm was complaining it was shadowing [the built-in name](https://docs.python.org/3/library/constants.html#license).